assets/queries/terraform/kubernetes/image_host_port_not_specified/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "4e74cf4f-ff65-4c1a-885c-67ab608206ce",
+  "queryName": "Workload Host Port Not Specified",
```

Comment:
You need to change the directory name too. To match the query name in the metadata.json

this should also be done here: https://github.com/Checkmarx/kics/tree/master/assets/queries/k8s/image_host_port_not_specified

To normalize our similar queries with the same name across platforms.